examples/medplum-chart-demo/src/components/clinical-chart/MedicationRequest.tsx

```diff
@@ -0,0 +1,85 @@
+import { Anchor, Badge, Box, Button, Group, Modal, Radio, Stack, Text } from '@mantine/core';
+import { useDisclosure } from '@mantine/hooks';
+import { getDisplayString } from '@medplum/core';
+import { CodeableConcept, MedicationRequest, Patient } from '@medplum/fhirtypes';
+import { CodeableConceptDisplay, CodeableConceptInput, Form, useMedplum } from '@medplum/react';
+import React, { useCallback, useState } from 'react';
+
+export interface MedicationRequestsProps {
+  patient: Patient;
+  medicationRequests: MedicationRequest[];
+}
+
+export function MedicationRequests(props: MedicationRequestsProps): JSX.Element {
+  const medplum = useMedplum();
+  const [medicationRequests, setMedicationRequests] = useState<MedicationRequest[]>(props.medicationRequests);
+  const [opened, { open, close }] = useDisclosure(false);
+  const [code, setCode] = useState<CodeableConcept>();
+
+  const handleSubmit = useCallback(
+    (formData: Record<string, string>) => {
+      console.log('handleSubmit', formData);
+      const status = formData.status as 'active' | 'stopped';
+      medplum
+        .createResource<MedicationRequest>({
+          resourceType: 'MedicationRequest',
+          status,
+          intent: 'order',
+          medicationCodeableConcept: code,
+          subject: {
+            reference: `Patient/${props.patient.id}`,
+            display: getDisplayString(props.patient),
+          },
+        })
+        .then((newRequest) => {
+          setMedicationRequests([newRequest, ...medicationRequests]);
+          close();
+        })
+        .catch(console.error);
+    },
+    [medplum, props.patient, medicationRequests, close, code]
+  );
+
+  return (
+    <>
+      <Group position="apart">
+        <Text fz="md" fw={700}>
+          Medication Requests
+        </Text>
+        <Anchor href="#" onClick={open}>
+          + Add
+        </Anchor>
+      </Group>
+      {medicationRequests.length > 0 ? (
+        <Box>
+          {medicationRequests.map((request) => (
+            <Badge mt={4} key={request.id} maw="50%" color={request.status === 'active' ? 'blue' : 'gray'}>
+              <CodeableConceptDisplay value={request.medicationCodeableConcept} />
+            </Badge>
+          ))}
+        </Box>
+      ) : (
+        <Text>(none)</Text>
+      )}
+      <Modal opened={opened} onClose={close} title="Add Medication Request">
+        <Form onSubmit={handleSubmit}>
+          <Stack h={275}>
+            <CodeableConceptInput
+              name="request"
+              data-autofocus={true}
+              binding="https://app.medplum.com/ValueSet/16d6f7b7-7eeb-4d0e-a83b-83be082aa10b"
```

Comment:
The valueset looks like this 

    "include": [
      {
        "system": "https://www.nlm.nih.gov/research/umls/rxnorm/index.html",
        "concept": [
          {
            "code": "1370977",
            "display": "Systemic HRT - Osphena"
          },
          {
            "code": "890917",
            "display": "Local HRT - Estrace"
          },
          {
            "code": "235988",
            "display": "SSRI/SNRI - Effexor"
          },
          {
            "code": "153010",
            "display": "OTC - Advil"
          }
        ]

Reply:
Is this the way we bind to Value Sets?  Do we not use `ValueSet.url` for the binding?

Reply:
We use the valueset.url for the binding. This is just an example of what the ValueSet resource shows